tests/snapshot_tests/snapshot_apps/scroll_visible.py

```diff
@@ -0,0 +1,22 @@
+from textual.app import App, ComposeResult
+from textual.containers import VerticalScroll
+from textual.widgets import Label, Static
+
+
+class MyCustomWidget(Static):
+    def compose(self) -> ComposeResult:
+        yield Label(("|\n" * 100)[:-1])
+        yield Label("CAN'T GET HERE", id="target")
```

Comment:
Can you change the text here? Also it's good to add a comment linking the test to the issue it fixes.